### DISCUSSION TITLE
Configurable templated title for MS Teams contact point

### DISCUSSION THREAD

**Discussion by ristoporila** [grafana/grafana#38526] [Alerting]
I am using Grafana 8 ngalerts and would like to configure titles for MS Teams notifications. This can be done for Slack contact points, but I can't find a way to do it for MS Teams.

**Comment by mucho-al-dente** [comment]
This has been addressed in e8f4b58a8ba40812c938942442ca8fb552a9360f and merged into `main`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
